hera_mc/cm_handling.py

```diff
@@ -154,35 +157,60 @@ def get_entry(self, session):
         elif pad > 0:
             self.keys_up.extend([None] * abs(pad))
 
+    def add_filter(self, at_date, rev_type):
+        # Set default to use it
```

Comment:
this comment doesn't clarify anything for me. Can you make a full docstring for this function? And it looks like whatever default you're talking about here isn't passed in, it's a property of the object, so please document how that comes in as well.